test/test_modeling_processor_saving_loading.py

```diff
@@ -25,17 +25,17 @@ def test_processor_saving_loading(caplog):
         train_filename="train-sample.json",
         dev_filename="dev-sample.json",
         test_filename=None,
-        data_dir=Path("samples/qa"),
+        data_dir=Path(__file__).parent/"samples"/"qa",
```

Comment:
I wonder if we could introduce a global variable for example `SAMPLES_DIR` in conftest.py that stores the samples path. This should be easier to read than always having `Path(__file__).parent`

Reply:
Good idea actually :+1: